packages/apputils-extension/style/contextualhelp.css

```diff
@@ -39,7 +31,12 @@ td {
   height: '2em';
 }
 
-.jp-ContextualShortcut-TableItem .jp-ContextualShortcut-Key {
+.jp-ContextualShortcut-Key {
+  font-family: var(--jp-code-font-family);
+
+  --jp-private-shortcuts-key-padding-horizontal: 5px;
+  --jp-private-shortcuts-key-padding-vertical: 3px;
```

Comment:
Should [relative length units](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#lengths) be used here so that for example when font size is changed the padding would also change?